### ISSUE TITLE
Unable to get CIRA to connect

### ISSUE THREAD

**Issue by rowds15** [Ylianst/MeshCentral#1787]
Hi I've migrated my Meshcentral server to an AWS instance and I'm not able to get any CIRA to connect back to the server, I also have an internal Meshcentral server which i can't seem to get the CIRA connect to either. I've tried WAN/LAN modes but I don't seem to see any traffic to either server in any mode.

I have used the CIRA meshscript, tried Manual Username/Password and also Manual Certificate. I don't seem to get any traffic back to the server on port 4433.

Not sure where to go to from here.

**Comment by jsastriawan** [comment]
Few checks to test
- If you are using FQDN, is this FQDN resolvable publicly to your AWS instance?
- Try accessing https://your_meshserver_address:4433/, does it respond with the some text informing that MPS is running? if not, you may need to check your AWS setting to open 4433 to your instance.

**Comment by drnikcuk** [comment]
Also, have you gone into trace and ran that for a while to see if anything is getting logged ? (not sure if these links are the same for installs but you will find it at https://yoururl/?viewmode=41 if they are)

**Comment by rowds15** [comment]
> 
> 
> Few checks to test
> 
>     * If you are using FQDN, is this FQDN resolvable publicly to your AWS instance?
> 
>     * Try accessing https://your_meshserver_address:4433/, does it respond with the some text informing that MPS is running? if not, you may need to check your AWS setting to open 4433 to your instance.

Yes I'm using the FQDN and am able to browse to https://meshserver:4433/

**Comment by rowds15** [comment]
> 
> 
> Also, have you gone into trace and ran that for a while to see if anything is getting logged ? (not sure if these links are the same for installs but you will find it at https://yoururl/?viewmode=41 if they are)

Yes ran tracing from Meshcentral console and also tcpdump on port 4433 no data received at all.
I have also tried this from another virtual instance internally on the LAN with the same result, No Data.

**Comment by rowds15** [comment]
hi drnikcuk, i have checked both servers and yes he url that the agent connect to is correct on both.
